universal-application-tool-0.0.1/app/services/applicant/Block.java

```diff
@@ -62,4 +62,8 @@ public boolean hasErrors() {
     }
     return errorsMemo.get();
   }
+
+  public boolean isValid() {
```

Comment:
heads up: [This PR](https://github.com/seattle-uat/universal-application-tool/pull/369) just added a `hasErrors` method here.

Reply:
Thanks I'll use that method instead!